packages/ngspice/meta.yaml

```
@@ -0,0 +1,40 @@
+package:
+  name: ngspice
+  version: "44.2"
+
```

```suggestion
package:
  name: ngspice
  version: "44.2"
  tag:
    - library
    - shared_library
```

Adding tags helps our build system to arrange the build order.